### ISSUE TITLE
"Parsing error: AST is missing the tokens array." when document contains 3 or more tables.

### ISSUE THREAD

**Issue by ricealexander** [yeonjuan/html-eslint#31]
When a document contains 3 or more tables, and none of those tables have `<tbody>` tags, the error "**Parsing error: AST is missing the tokens array.**" is displayed.

I am using the recommended configuration:
<details open>
<summary>.eslintrc.js</summary>
<br>

```js
module.exports = {
  plugins: ['@html-eslint'],
  overrides: [
    {
      files: ['*.html'],
      parser: '@html-eslint/parser',
      'extends': ['plugin:@html-eslint/recommended'],
    },
  ],
}
```

</details>

This code throws the AST is missing tokens error:

```html
<!DOCTYPE html>
<html lang="en">
</head>
<body>

  <table>
    <tr>
      <td>Table 1</td>
    </tr>
  </table>

  <table>
    <tr>
      <td>Table 2</td>
    </tr>
  </table>

  <table>
    <tr>
      <td>Table 3</td>
    </tr>
  </table>

</body>
</html>
```

This code does not throw the AST tokens error and correctly reports @html-eslint/indent (only 2 tables)

```html
<!DOCTYPE html>
<html lang="en">
</head>
<body>

  <table>
    <tr>
      <td>Table 1</td>
    </tr>
  </table>

  <table>
    <tr>
      <td>Table 2</td>
    </tr>
  </table>

</body>
</html>
```

This code does not throw the AST tokens error and correctly reports @html-eslint/indent (tables have tbody)
```html
<!DOCTYPE html>
<html lang="en">
</head>
<body>

  <table>
    <tbody>
      <tr>
        <td>Table 1</td>
      </tr>
    </tbody>
  </table>

  <table>
    <tbody>
      <tr>
        <td>Table 2</td>
      </tr>
    </tbody>
  </table>

  <table>
    <tbody>
      <tr>
        <td>Table 3</td>
      </tr>
    </tbody>
  </table>

</body>
</html>
```

**Comment by yeonjuan** [comment]
Thanks a lot! There was a critical bug in the ast-converter. #36 will fix it :)
